src/software/world/robot_state.h

```diff
@@ -77,3 +82,12 @@ class RobotState
     Angle orientation_;
     AngularVelocity angular_velocity_;
 };
+
+/**
+ * A light structure for a robot state with an ID
+ */
+struct RobotStateWithId
+{
+    unsigned int id;
```

Comment:
we can move the `using RobotId` from robot.h to robot_state.h and use `RobotId` here and through out physicsworld.

Reply:
done

Reply:
```suggestion
    RobotId id;
```
looks like you missed this one^